packages/core/types/src/workflow/fulfillment/create-fulfillment.ts

```diff
@@ -180,4 +180,7 @@ export type CreateFulfillmentWorkflowInput = {
    * The associated fulfillment order.
    */
   order: CreateFulfillmentOrderWorkflowDTO
+
+  // TODO: revisit - either remove `order_id` or `order`
+  order_id: string
```

Comment:
thought(non-blocking): I think we should call `order` something different. It is really just additional context that you can pass upon creating a fulfillment. But not super important for this PR.